projects/packages/external-media/src/shared/sources/openverse/style.scss

```diff
@@ -0,0 +1,18 @@
+.jetpack-external-media-header__openverse {
```

Comment:
Separated the styles from the [editor/editor.scss](https://github.com/Automattic/jetpack/pull/41282/files#diff-be7109b2a4cce65657c706fe151f7b27b53a17cab32d7eff0571987f8959fbd2)